qemu/tests/cfg/usbdev.cfg

```diff
@@ -0,0 +1,151 @@
+                    usb_port_d5 = "2.6"
+                    usb_port_d6 = "2.7"
+                    usb_port_d7 = "2.8"
+
+    # usb devices
+    variants:
+        - usb_kbd:
+            usb_type = "usb-kbd"
+            info_usb_name = "QEMU USB Keyboard"
+            vp_id = "0627:0001"
+            vendor = "Adomax Technology Co., Ltd"
+            product = "QEMU USB Keyboard"
+            usb_devices += " testdev"
+        - usb_mouse:
+            usb_type = "usb-mouse"
+            info_usb_name = "QEMU USB Mouse"
+            vp_id = "0627:0001"
+            vendor = "Adomax Technology Co., Ltd"
+            product = "QEMU USB Mouse"
+            usb_devices += " testdev"
+        - usb_tablet:
+            usb_type = "usb-tablet"
+            info_usb_name = "QEMU USB Tablet"
+            vp_id = "0627:0001"
+            vendor = "Adomax Technology Co., Ltd"
+            product = "QEMU USB Tablet"
+            usb_devices += " testdev"
+        - usb_ccid:
+            usb_type = "usb-ccid"
+            info_usb_name = "QEMU USB CCID"
+            vp_id = "08E6:4433"
+            vendor = "Gemplus"
+            product = "QEMU USB CCID"
+            usb_devices += " testdev"
+        - usb_hub:
+            usb_type = "usb-hub"
+            info_usb_name = "QEMU USB Hub"
+            vp_id = "0409:55aa"
+            vendor = ""
+            product = "QEMU USB Hub"
+            usb_devices += " testdev"
+        - usb_storage:
+            usb_type = "usb-storage"
+            info_usb_name = "QEMU USB MSD"
+            vp_id = "46f4:0001"
+            vendor = ""
+            product = "QEMU USB HARDDRIVE"
+            no piix3-uhci piix4-uhci ich9-uhci
+            max_usb_dev:
+                usb_devices = " tablet1 hub1"
+            udevs += " d0"
+            start_vm = no
+        - usb_bot:
+            usb_type = "usb-bot"
+            info_usb_name = "QEMU USB MSD"
+            vp_id = "46f4:0001"
+            vendor = ""
+            product = "QEMU USB HARDDRIVE"
+            no piix3-uhci piix4-uhci ich9-uhci
+            max_usb_dev:
+                usb_devices = " tablet1 hub1"
+            udevs += " d0"
+            start_vm = no
+    # usb tests
+    variants:
+        - check:
+            type = usbdev_check
+        - hotplug:
+            no RHEL.3 RHEL.4 RHEL.5
+            no usb_bot
+            type = usbdev_hotplug
+            only without_usb_hub
+            usb_devices= " "
+            start_vm = yes
+            repeat_times = 1
+        - migration:
+            mig_timeout = 3600
+            mig_protocol = "tcp"
+            type = usbdev_migration
+        - function:
+            type = usbdev_func
+            variants:
+                - io_without_fs:
+                    only usb_storage usb_bot
+                    function = io_without_fs
+                - io_with_fs:
+                    only usb_storage usb_bot
+                    function = io_with_fs
```

Comment:
@Zhengtong they have been added as single patch,and you can find them at:
https://github.com/autotest/tp-qemu/pull/710
https://github.com/autotest/tp-qemu/pull/709
https://github.com/autotest/tp-qemu/pull/708